# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/WebrtcMediaDataDecoderCodec.h
# Commit: 709078257c62
# Full Hash: 709078257c628fae054382894de5864b1f5f5c2a
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 3
# Description:
#   Bug 1654112 - Re-wire GMP PluginID between codecs and VideoConduit. r=bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104724
# ==============================================================================

diff -r 3ca24e761cc3 -r 709078257c62 dom/media/webrtc/libwebrtcglue/WebrtcMediaDataDecoderCodec.h
--- a/dom/media/webrtc/libwebrtcglue/WebrtcMediaDataDecoderCodec.h	Wed Feb 10 10:34:14 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/WebrtcMediaDataDecoderCodec.h	Wed Feb 10 15:17:29 2021 +0100
@@ -31,9 +31,6 @@
  public:
   explicit WebrtcMediaDataDecoder(nsACString& aCodecMimeType);
 
-  // Implement VideoDecoder interface.
-  uint64_t PluginID() const override { return 0; }
-
   int32_t InitDecode(const webrtc::VideoCodec* codecSettings,
                      int32_t numberOfCores) override;
 